content/docs/ref/pr.md

```diff
@@ -58,6 +62,13 @@ The `--merge`, `--rebase`, and `--squash` options enable
 (GitLab) to merge the pull request as soon as checks succeed. If waiting for
 checks isn't supported, `cml pr` will try to merge the pull request immediately.
 
+
+### Custom options example
+
+```cli
+$ cml pr --title="Periodic Retaining" --body="$(cat ./report.md)" --branch="$(date +%Y-%M)" .
+```
+
```

Comment:
do we really need this example?

Reply:
Do we need it? No. Do you feel this example will pull users to an unnecessary (for the average user) complex usage that might be hurtful? I feel fine saying this is a [sustainable example](https://github.com/iterative/cml-playground/blob/main/.github/workflows/cml-1063.yml#L80)

Reply:
seemed to not add much to the existing `Options` description... But fine to resolve unresolved by me.

Reply:
perhaps a quick @iterative/docs opinion?

Reply:
> pull users to an unnecessary (for the average user) complex usage

At first glance at least, most examples here could be deemed overcomplicated (at least lacking down-to-earth explanations), so this would be in line with that. Up to you on how many Examples you'd like to maintain in the references.